include/swift/SILOptimizer/Utils/SILInliner.h

```diff
@@ -36,23 +36,38 @@ enum class InlineCost : unsigned {
 InlineCost instructionInlineCost(SILInstruction &I);
 
 class SILInliner : public TypeSubstCloner<SILInliner> {
-public:
   friend class SILInstructionVisitor<SILInliner>;
   friend class SILCloner<SILInliner>;
-  
-  enum class InlineKind {
-    MandatoryInline,
-    PerformanceInline
-  };
 
+public:
+  enum class InlineKind { MandatoryInline, PerformanceInline };
+
+private:
+  InlineKind IKind;
+
+  SILBasicBlock *CalleeEntryBB = nullptr;
+
+  /// \brief The location representing the inlined instructions.
+  ///
+  /// This location wraps the call site AST node that is being inlined.
+  /// Alternatively, it can be the SIL file location of the call site (in case
+  /// of SIL-to-SIL transformations).
+  Optional<SILLocation> Loc;
+  const SILDebugScope *CallSiteScope = nullptr;
+  SILFunction *CalleeFunction = nullptr;
+  llvm::SmallDenseMap<const SILDebugScope *, const SILDebugScope *>
```

Comment:
What is the default here? Can you just put that in so that when reading the code it is obvious?

Reply:
Which default?

Reply:
The default small size of the default dense map.